conda-forge.yml

```
@@ -1,3 +1,11 @@
+azure:
+  free_disk_space: true
+  store_build_artifacts: true
```

This comment was dropped due to the bot's rearranging of this file

```suggestion
  # Enable artifact storage to test with GPUs locally
  store_build_artifacts: true
```